frr-reloader/frr-reloader.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+cleanup() {
```

Comment:
Neat! I am probably biased by my weak bash knowledge, any reason why we did not go with a go process?